tests/workspace/test_manager.py

```diff
@@ -48,8 +48,12 @@ def _get_ws_manager(mock_connection, pdf_content=MOCK_SNOWFLAKE_YML_FILE):
 
 
 def test_pdf_not_v2(temp_dir):
+    pdfv1 = PdfV10Factory.build(
+        native_app__source_stage="app_src.stage",
+        native_app__artifacts=[{"src": "app/*", "dest": "./"}],
+    )
     with pytest.raises(InvalidProjectDefinitionVersionError):
-        _get_ws_manager(pdf_content=MOCK_SNOWFLAKE_YML_V1_FILE)
+        _get_ws_manager(pdf_content=str(pdfv1))
```

Comment:
not a fan of this pattern. If we need JSON, then make it explicit: `pdfv1.as_json_str()` or something like that. IMHO that's much better for maintainability in the future.

Reply:
done